packages/store/src/storage.ts

```diff
@@ -1127,9 +1127,12 @@ class Storage {
 
     // 1. after tarball has been successfully uploaded, we update the version
     try {
-      // TODO: review why do this
-      versions[versionToPublish].readme =
-        _.isNil(manifest.readme) === false ? String(manifest.readme) : '';
+      // Older package managers like npm6 do not send readme content as part of version but include it on root level
```

Comment:
that's actually true, I had this logic but maybe somewhere but not here, thanks.